build/x86_64_sles15/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM opensuse/leap:15.1
+
+RUN zypper -n refresh \
+ && zypper -n install \
+        autoconf \
+        bzip2 \
+        curl \
+        expect \
+        gcc-c++ \
+        git \
+        patch \
+        ruby \
+        rpm-build \
+        which \
+        zlib-devel \
+  # RVM GPG keys
+  && gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
+  && (curl -sSL https://get.rvm.io | /bin/bash -s stable) \
+  # RVM requires running in a login shell.
+  && /bin/bash -l -c "rvm requirements && rvm install 2.5.3 && gem install bundler --no-document && gem update" \
+  && zypper -n clean
```

Comment:
Can you please make sure that the content of `/etc/SuSE-release` and `/etc/os-release` is identical to that of a raw SLES 15 machine? Thanks.

Reply:
1adb154e56f1:/ # cat /etc/os-release 
NAME="openSUSE Leap"
VERSION="15.1 "
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.1"
PRETTY_NAME="openSUSE Leap 15.1"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.1"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
1adb154e56f1:/ # cat /etc/SuSE-release
cat: /etc/SuSE-release: No such file or directory

sophieyfang@sophieyfang-google-fluentd-sles-15-vm:~> cat /etc/os-release
NAME="SLES"
VERSION="15-SP1"
VERSION_ID="15.1"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP1"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp1"
sophieyfang@sophieyfang-google-fluentd-sles-15-vm:~> cat /etc/SuSE-release
cat: /etc/SuSE-release: No such file or directory